grafana-plugin/src/models/alertgroup/alertgroup.types.ts

```diff
@@ -85,6 +85,8 @@ export interface Alert {
   alert_receive_channel: Partial<AlertReceiveChannel>;
   paged_users: PagedUser[];
   team: GrafanaTeam['id'];
+  grafana_incident_id: string | null;
+  has_a_related_grafana_incident: boolean;
```

Comment:
Wondering if it is worth it having the boolean property in this case instead of just checking if `grafana_incident_id` is set.

Reply:
fair point. thought it could be a useful helper attribute but let's start simpler for now 👍